--- d4x-2.5.0rc2/share/nt.desktop~	Wed Jul 28 00:01:20 2004
+++ d4x-2.5.0rc2/share/nt.desktop	Wed Jul 28 00:01:20 2004
@@ -1,18 +1,21 @@
 [Desktop Entry]
 Name=Downloader for X
-Name[ru]=Качалка для Иксов
-Name[fr]=Downloader for X
+Name[de]=Downloader fц╪r X
 Name[es]=Downloader para X
-Name[de]=Downloader fЭr X
+Name[fr]=Downloader for X
 Name[pl]=Downloader dla X
-Comment=downloading program
-Comment[ru]=программа для скачивания файлов
-Comment[fr]=Programme de tИlИchargement de fichiers
-Comment[es]=Una herramienta para descagar ficheros
+Name[ru]=п п╟я┤п╟п╩п╨п╟ п╢п╩я▐ п≤п╨я│п╬п╡
+Comment=Downloading program
 Comment[de]=Programm zum Downloaden von Dateien
-Comment[pl]=Program do pobierania plikСw z Internetu.
+Comment[es]=Una herramienta para descagar ficheros
+Comment[fr]=Programme de tц╘lц╘chargement de fichiers
+Comment[pl]=Program do pobierania plikцЁw z Internetu
+Comment[ru]=п©я─п╬пЁя─п╟п╪п╪п╟ п╢п╩я▐ я│п╨п╟я┤п╦п╡п╟п╫п╦я▐ я└п╟п╧п╩п╬п╡
 TryExec=nt
 Exec=nt
 Icon=nt.png
-Terminal=0
+Terminal=false
 Type=Application
+Categories=GTK;Network;FileTransfer;
+Encoding=UTF-8
+# vi: encoding=utf-8
