diff -urN d4x-2.4beta2.orig/share/Makefile.am d4x-2.4beta2/share/Makefile.am
--- d4x-2.4beta2.orig/share/Makefile.am	Wed Oct 30 23:44:27 2002
+++ d4x-2.4beta2/share/Makefile.am	Wed Oct 30 23:44:42 2002
@@ -21,7 +21,7 @@
 #
 install-data-local:
 	for i in $(SOUNDFILES) $(THEMEFILES); do \
-	    $(INSTALL_DATA) -D $$i $(pkgdatadir)/`dirname $$i | sed 's,$(srcdir)/,,'`/`basename $$i`; \
+	    $(INSTALL_DATA) -D $$i $(DESTDIR)$(pkgdatadir)/`dirname $$i | sed 's,$(srcdir)/,,'`/`basename $$i`; \
 	done
 
 uninstall-local:
