diff -urN aaa/d4x-2.4.1/main/Makefile.am d4x-2.4.1/main/Makefile.am
--- aaa/d4x-2.4.1/main/Makefile.am	Mon Jan 27 05:40:07 2003
+++ d4x-2.4.1/main/Makefile.am	Mon Jan 27 21:56:23 2003
@@ -2,8 +2,6 @@
 # $Header$
 #
 
-include $(top_srcdir)/admin/Makefile.addons
-
 SUBDIRS = face
 
 MAINTAINERCLEANFILES = Makefile.in
diff -urN aaa/d4x-2.4.1/main/face/Makefile.am d4x-2.4.1/main/face/Makefile.am
--- aaa/d4x-2.4.1/main/face/Makefile.am	Mon Jan 27 05:40:09 2003
+++ d4x-2.4.1/main/face/Makefile.am	Mon Jan 27 21:56:40 2003
@@ -2,8 +2,6 @@
 # $Header$
 #
 
-include $(top_srcdir)/admin/Makefile.addons
-
 MAINTAINERCLEANFILES = Makefile.in
 CLEANFILES = *~
 
