--- main/sndserv.cc~	Thu Mar 28 10:40:39 2002
+++ main/sndserv.cc	Thu Jul 18 10:37:07 2002
@@ -135,7 +135,7 @@
 
 #if G_BYTE_ORDER == G_LITTLE_ENDIAN
 	*len =(buf[3] << 24) | (buf[2] << 16) | (buf[1] << 8) | buf[0];
-#elif
+#else
 	*len =(buf[0] << 24) | (buf[1] << 16) | (buf[2] << 8) | buf[3];
 #endif
 
@@ -148,7 +148,7 @@
 		return 0;
 #if G_BYTE_ORDER == G_LITTLE_ENDIAN
 	*val = (buf[1] << 8) | buf[0];
-#elif
+#else
 	*val = (buf[0] << 8) | buf[1];
 #endif
 	return 1;
